fluid-project/handbook.floeproject.org
# feat: add article contents to article/topic pages

## Description

This update adds the Article Contents accordion control to all article/topic pages within the site.

## Steps to test

1. Go to _Perspectives_
2. Click on "Achieving Inclusive Education" or its thumbnail image
3. Click on "Article Contents"

**Expected behavior:** The accordion contents drop-down should be present and should contain a link to each level 2 heading (h2) on the page (there are two on that page).

## Additional information

Styling the open/closed arrow icon is tricky when using the `details` and `summary` elements, so they do not quite match the design.

## Related issues

Resolves #59
